.gitignore

```diff
@@ -329,4 +329,6 @@ tools/
 
 # VisualStudioCode Patch
 # Ignore all local history of files
-.history
\ No newline at end of file
+.history
+/Dump
```

Comment:
Ignoring the "dump" files created when troubleshooting builds using the [`DumpXml` utility](https://github.com/nunit/nunit3-vs-adapter/blob/bddee2c7739f9cc91a021aadf9b3a4f2ddba4c2c/src/NUnitTestAdapter/DumpXml.cs#L42).